proposals/multitenancy.md

```diff
@@ -0,0 +1,63 @@
+# Multitenancy
```

Comment:
should we remove this doc? It's confusing to keep it imo.

Reply:
What I have seen in the past is that proposals are not generally removed even after implementation. It can be nice to understand why a particular path was not pursued.